modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerContainerFragment.kt

```diff
@@ -84,16 +84,24 @@ class PlayerContainerFragment : BaseFragment(), HasBackstack {
 
         val binding = binding ?: return
 
```

Comment:
Since now we always use embedded `UpNextFragment`, can we check use of `embedded` param here:

`val upNextFragment = UpNextFragment.newInstance(embedded = true, source = UpNextSource.NOW_PLAYING)`

It seems to be used in the `UpNextFragment` for styling status and nav bars but I cannot make out any difference  specially after the recent Up Next theming changes in https://github.com/Automattic/pocket-casts-android/pull/1605.

Reply:
Good point! I'll take a look at it before merging.